web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@pedrobaeza It's not the same icon ! https://github.com/OCA/web/pull/128/files?short_path=e340032#diff-e34003280f7857fb9036ec1c6f49edc0

regards.